cl8-web/package.json

```diff
@@ -11,7 +11,7 @@
     "lint": "vue-cli-service lint",
     "deploy": "firebase deploy --only hosting",
     "release": "vue-cli-service build && npm run deploy && npm run notify",
-    "notify": "node ./server/scripts/notifySlackOfDeploy.js"
+    "notify": "# node ./server/scripts/notifySlackOfDeploy.js"
```

Comment:
Temporary stuff?

Reply:
Yeah.

By default this would deploy to a slack channel, so people can see when deploys have happened.

It's not so clear where send notifications yet, so I'm holding off for now., as we have two separate slack channels.